osu.Server.Queues.ScoreStatisticsProcessor/Commands/Maintenance/VerifyImportedScoresCommand.cs

```diff
@@ -188,6 +189,13 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
                             sqlBuffer.Append($"UPDATE scores SET pp = {importedScore.HighScore.pp.ToString() ?? "NULL"} WHERE id = {importedScore.id};");
                         }
 
+                        if (!check(importedScore.id, "ranked", importedScore.ranked, true))
```

Comment:
I guess this implicitly assumes that every score in the high tables is automatically ranked, but what about the `hidden` flag in those tables? Are we handling that elsewhere already? I don't think we are, or at least I can't see it happening in code...

Reply:
Hidden is exclusively used for restricted users, which is handled as part of the ES indexing step these days. So for the time being we do just import these scores to the new table with no consideration of the flag, and leave that to [es indexer](https://github.com/ppy/osu-elastic-indexer/blob/50f2ab5fe6846d5ad148b6080477fb55ba67ea33/osu.ElasticIndexer/Score.cs#L33) to get right.